freqtrade/freqtradebot.py

```diff
@@ -443,33 +442,29 @@ def process_maybe_execute_buy(self) -> None:
         except DependencyException as exception:
             logger.warning('Unable to create trade: %s', exception)
 
-    def process_maybe_execute_sell(self, trade: Trade) -> bool:
+    def process_maybe_execute_sells(self, trades: List[Any]) -> None:
         """
-        Tries to execute a sell trade
-        :return: True if executed
+        Tries to execute sell trades in a safe way
```

Comment:
Tries to sell trades in a safe way (?) 

the "execute" seems misplaced now

Reply:
I guess the word 'trades' is misplaced here (and in the docstring for _buy() counterpart). The `process_maybe_execute_sells` executes sells (sell orders), same for `process_maybe_execute_buys`-- it executes buys (buy orders)...

Reply:
actually it checks trades (so trades is correct i think) for sell (or buy) reasons and eventually places a sell-order if necesary ... (it doesn't blindly execute a sell-order, that's what `execute_sell()` does.

Reply:
Changed docstrings, pls review.